administrator/components/com_installer/src/Field/PackageField.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_installer
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Installer\Administrator\Field;
+
+use Joomla\CMS\Form\Field\ListField;
+use Joomla\Component\Installer\Administrator\Helper\InstallerHelper;
+
+/**
+ * Package field.
+ *
+ * Selects the extension ID of an extension of the "package" type.
+ *
+ * @since __DEPLOY_VERSION__
+ */
+class PackageField extends ListField
+{
+	/**
+	 * Method to get the field options.
+	 *
+	 * @return  array  The field option objects.
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected function getOptions()
+	{
+		$options = InstallerHelper::getPackageOptions();
+
+		return array_merge($options, parent::getOptions());
```

Comment:
```suggestion
		return array_merge(parent::getOptions(), $options);
```
![image](https://user-images.githubusercontent.com/66922325/169629535-cad4d8d2-c3f6-4c60-8cea-3019f19a0d4f.png)

Reply:
I am not going to accept that on the grounds of maintaining consistency with the other filter fields of com_users and Joomla in general. Joomla puts the no selection option first on the list. Regardless of our personal preference, that's how Joomla does it, that's how we have to do it in our PRs.

Reply:
Sorry, the screenshot is before the suggested change 😅

Here is for example the FolderField:
https://github.com/joomla/joomla-cms/blob/444965e0b7d5f0e393691d86d9f634989ebad383/administrator/components/com_installer/src/Field/FolderField.php#L43

Reply:
Ah, that makes more sense to me now 😄 OK, I will commit this change.